pkg/securitypolicy/securitypolicy.go

```diff
@@ -3,108 +3,25 @@ package securitypolicy
 import (
 	"encoding/base64"
 	"encoding/json"
-	"regexp"
 	"strconv"
 
 	"github.com/pkg/errors"
 )
 
-type EnvVarRule string
+type EnvRuleStrategy string
 
 const (
-	EnvVarRuleString EnvVarRule = "string"
-	EnvVarRuleRegex  EnvVarRule = "re2"
+	EnvRuleString EnvRuleStrategy = "string"
+	EnvRuleRegex  EnvRuleStrategy = "re2"
 )
 
-// PolicyConfig contains toml or JSON config for security policy.
```

Comment:
Can you break out the config refactoring into another PR?

Reply:
done